api/helpers/models/projectNotification.js

```diff
@@ -5,12 +5,16 @@ module.exports = require('../models')('ProjectNotification', {
   name: { type: String, default: null },
   type: { type: String, default: null },
   subType: { type: String, default: null },
+  proponent: { type: String, default: null },
   nature: { type: String, default: null },
   region: { type: String, default: null },
   location: { type: String, default: null },
   decision: { type: String, default: null },
   decisionDate: { type: Date, default: null },
+  notificationReceivedDate: { type: Date, default: null },
   description: { type: String, default: null },
+  notificationThresholdValue: { type: String, default: null },
```

Comment:
According to AC, this should only accept numeric vals

Reply:
good catch,  updated